CLI tools/MultiMC to Git.py

```diff
@@ -14,8 +15,29 @@ def copy_file(from_path, to_path, from_desc, to_desc):
     else:
         print("Skipped " + from_desc + " copying to " + to_desc + ", didn't exist")
 
+def copy_to_archive(from_path, to_path, archive_path):
+
+    files = []
+
+    with zipfile.ZipFile(archive_path) as archive:
+        for zipinfo in archive.infolist():
+            if zipinfo.filename != to_path:
+                files.append((zipinfo.filename, archive.read(zipinfo.filename)))
+
+    with zipfile.ZipFile(archive_path, "w") as archive:
+        for filename, content in files:
+            archive.writestr(filename, content)
+        archive.write(from_path, to_path)
+
 # MultiMC to Git
 
-copy_file(mmc_path + "instance.cfg", git_path + "MultiMC\Fabulously Optimized x.y.z/instance.cfg", "MultiMC instance.cfg", "Git")
-copy_file(mmc_path + "mmc-pack.json", git_path + "MultiMC\Fabulously Optimized x.y.z/mmc-pack.json", "MultiMC mmc-pack.json", "Git")
-copy_file(mmc_path + "pack.png", git_path + "MultiMC\Fabulously Optimized x.y.z/pack.png", "MultiMC pack.png", "Git")
\ No newline at end of file
+file = glob.glob("Desktop/Fabulously Optimized-*.zip", root_dir=user_path)[0]
+version = "-".join(file.replace(".zip", "").split("-")[1:])

```

Comment:
Could you actually get the version number from `name=` in file `mmc_path + "instance.cfg"`? I think that'd match my workflow better